assets/js/modules/thank-with-google/components/settings/SettingsView.js

```diff
@@ -75,6 +79,29 @@ export default function SettingsView() {
 		return null;
 	}
 
+	let supporterWall;
+
+	if ( supporterWallSidebars === undefined ) {
+		supporterWall = <ProgressBar small />;
+	} else if ( supporterWallSidebars.length > 0 ) {
+		supporterWall = (
+			<p className="googlesitekit-settings-module__meta-item-data">
+				<DisplaySetting value={ supporterWallSidebars?.join( ', ' ) } />
```

Comment:
No need for `?.` due to the check for `supporterWallSidebars.length` above

```suggestion
				<DisplaySetting value={ supporterWallSidebars.join( ', ' ) } />
```